Resources/ConfigPresets/L5/L5.toml

```diff
@@ -69,9 +72,10 @@ tickrate        = 30
 limit = 10.0
 
 [admin]
-see_own_notes           = true
-deadmin_on_join         = true
-allow_multi_server_play = true
+see_own_notes               = true
```

Comment:
Not super relevant at this point, but this was a disaster upstream and we should probably disable it if we ever get to the point where we stop giving every player admin access.

Reply:
I would like to. The bawwing over the slightest note upstream was absolutely miserable from an administrative perspective, and I would like to avoid a repeat of that on Lagrange.